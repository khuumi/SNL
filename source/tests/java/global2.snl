recipe print_1:
	start print1:
		do show to a
	done
done

recipe print_2:
	start print2:
		do show to b
	done
done

recipe increment:
	start incr:
		a is a + 1
		b is b + 1
	done
done

start main:
	a is 42
	b is 24
	do print_1
	do print_2
	do increment
	do print_1
	do print_2
done
