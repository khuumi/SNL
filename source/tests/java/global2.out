42
24
43
25
